webapp-django/crashstats/profile/views.py

```diff
@@ -0,0 +1,100 @@
+import datetime
+
+from django import http
+from django.contrib.auth.decorators import login_required
+from django.contrib.sites.models import RequestSite
+from django.shortcuts import render, redirect
+from django.db import transaction
+from django.contrib.auth.models import Permission
+
+from crashstats.crashstats.decorators import pass_default_context
+from crashstats.supersearch.models import SuperSearchUnredacted
+from crashstats.tokens import models
+from crashstats.tokens import forms
+
+
+@pass_default_context
+@login_required
+@transaction.atomic
+def profile(request, default_context=None):
+    context = default_context or {}
+    context['permissions'] = (
+        Permission.objects.filter(content_type__model='')
+        .order_by('name')
+    )
+
+    one_month_ago = (
```

Comment:
Ideally you don't want to be tied to a value in a variable name. Can you call it something else? Also, 4 weeks is not a month. The concept of "One month ago" can't exist without context.